jwst/datamodels/schemas/pathlosscorr.schema.yaml

```diff
@@ -1,34 +1,8 @@
 type: object
 properties:
-  pathloss_pointsource2d:
-    title: 2-d array for pathloss (point source)
-    fits_hdu: PATHLOSS_POINTSOURCE2D
+  pathloss:
+    title: 2D array for pathloss
```

Comment:
Minor nit, but for consistency with title of barshadow extension, which is "Bar shadow correction", why don't we make this "Path loss correction".